modules/gapi/src/backends/fluid/gfluidcore_func.simd.hpp

```diff
@@ -81,6 +81,9 @@ MUL_SIMD(float, float)
 
 #undef MUL_SIMD
 
+struct add_op {};
```

Comment:
simple `find on a page` doesn't return to me anything about using this structure: where is it used?

Reply:
Ok. Removed.